### PR TITLE
feat/create post with tags

### PR DESCRIPTION
**Pull request by YoshkiMatryoshki**
tag creation and post_tag merging logic implementation

### CODE REVIEW

File: blog-server-api/src/endpoints/create_post/handler.rs [hunk 1]
```diff
@@ -14,14 +14,28 @@ pub async fn http_handler(
...
-    let base_post = new_post_data
-        .map_err(|e| ValidationError {
-            reason: e.to_string(),
-        })?
-        .into(author.id);
+    let base_post = new_post_data.map_err(|e| ValidationError {
+        reason: e.to_string(),
+    })?;
+
+    let tag_tittles: Vec<String> = base_post.tags.iter().map(|t| t.title.clone()).collect();
```
Comment (tikitko) [inline blog-server-api/src/endpoints/create_post/handler.rs:21]: tit`t`les?

File: blog-server-services/src/lib.rs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 pub mod impls;
 pub mod traits;
+pub mod utility;
```
Comment (tikitko) [inline blog-server-services/src/lib.rs:3]: discuss?

**Review by tikitko — CHANGES_REQUESTED** [review]

File: blog-server-api/src/endpoints/create_post/handler.rs (hunk 1 above)
Comment (YoshkiMatryoshki) [reply blog-server-api/src/endpoints/create_post/handler.rs:21]: 🤡

File: blog-server-services/src/lib.rs (hunk 2 above)
Comment (YoshkiMatryoshki) [reply blog-server-services/src/lib.rs:3]: +

File: blog-server-api/src/endpoints/create_post/handler.rs (hunk 1 above)
Comment (YoshkiMatryoshki) [reply blog-server-api/src/endpoints/create_post/handler.rs]: done

File: blog-server-services/src/lib.rs (hunk 2 above)
Comment (YoshkiMatryoshki) [reply blog-server-services/src/lib.rs]: some magic happened

**Review by tikitko — APPROVED** [review]

**Pull request merged**